aports is a git tree so these git commands get the wrong directory
--- a/tools/gdb/gdbinit
+++ b/tools/gdb/gdbinit
@@ -50,17 +50,7 @@
 
 def set_src_dir(compile_dir):
   global src_dir
-  git = subprocess.Popen(
-      ['git', '-C', compile_dir, 'rev-parse', '--show-toplevel'],
-      stdout=subprocess.PIPE,
-      stderr=subprocess.PIPE)
-  src_dir, _ = git.communicate()
-  if git.returncode:
-    return
-  if isinstance(src_dir, str):
-    src_dir = src_dir.rstrip()
-  else:
-    src_dir = src_dir.decode('utf-8').rstrip()
+  src_dir = os.path.abspath(os.getcwd())
 
   load_libcxx_pretty_printers(src_dir)
 
